Resources/Prototypes/Entities/Structures/Wallmounts/shotgun_cabinet.yml

```diff
@@ -0,0 +1,83 @@
+- type: entity
+  id: ShotGunCabinet
+  name: shotgun cabinet
+  description: There is a small label that reads "For Emergency use only" along with details for safe use of the shotgun. As if.
+  components:
+  - type: MeleeSound
+    soundGroups:
+      Brute:
+        path:
+          "/Audio/Effects/glass_hit.ogg"
+  - type: WallMount
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Sprite
+    sprite: Structures/Wallmounts/shotgun_cabinet.rsi
+    layers:
+    - state: cabinet
+    - state: shotgun
+      map: ["enum.ItemCabinetVisualLayers.ContainsItem"]
+      visible: true
+    - state: glass
+      map: ["enum.ItemCabinetVisualLayers.Door"]
+  - type: ItemCabinet
+    cabinetSlot:
+      ejectOnInteract: true
+      whitelist:
+        tags:
+        - WeaponShotgunKammerer
+    doorSound:
+      path: /Audio/Machines/machine_switch.ogg
+    openState: glass-up
+    closedState: glass
+  - type: Appearance
+  - type: Lock
+  - type: AccessReader
+    access: [["Security"], ["Command"]]
+  - type: ItemSlots
+  - type: ContainerContainer
+    containers:
+      ItemCabinet: !type:ContainerSlot
+  placement:
+    mode: SnapgridCenter
+
+- type: entity
+  id: ShotGunCabinetOpen
+  parent: ShotGunCabinet
+  suffix: Open
+  components:
+  - type: ItemCabinet
+    opened: true
+    doorSound:
+      path: /Audio/Machines/machine_switch.ogg
+    openState: glass-up
+    closedState: glass
+
+- type: entity
+  id: ShotGunCabinetFilled
+  parent: ShotGunCabinet
+  suffix: Filled
+  components:
+  - type: ItemCabinet
+    cabinetSlot:
+      startingItem: WeaponShotgunKammerer
```

Comment:
Security outposts could be looted by powergamers. It probably should be at least double-barreled shotgun, yet it's still be powergame-able. This should not be an issue with these cabinets in brig (i.e. warden office).